client/src/pages/Profile/RequestCard/RequestCard.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import { Paper, Typography, Rating, Button, Stack, TextField } from '@mui/material';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DateTimePicker from '@mui/lab/DateTimePicker';
+
+export default function RequestCard(): JSX.Element {
+  const hourlyRate = 14;
+  const rating = 4;
+  const [start, setStart] = useState<Date | null>(new Date());
+  const [end, setEnd] = useState<Date | null>(new Date());
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
We can move the `LocalizationProvider` to the App.tsx so that it can be reused across the app